### ISSUE TITLE
[PRO]: The unique-id extension breaks the youtube extension

### ISSUE THREAD

**Issue by Haraldson** [ueberdosis/tiptap#4652]
### Which packages did you experience the bug in?

unique-id

### What Tiptap version are you using?

core 2.1.12, unique-id 2.4.1

### What’s the bug you are facing?

We have a concept where every heading, paragraph, list and list item get their own unique IDs in order to identify and diff content between “templates” and content derived from those.

Thus, we have configured `unique-id` like follows:

```javascript
UniqueID.configure({
    types: [
        'heading', 'paragraph',
        'bulletList', 'orderedList', 'listItem'
    ]
})
```

The problem occurs when trying to combine this with the `youtube` extension, which – as you probably know – is based on pasting video URLs directly into the content. From which node type will that paste action most likely happen? You guessed it: `paragrah` – a `unique-id`-affected node type.

The `youtube` extension immediately starts working, and the error message (outlined below) completely goes away, when I remove `paragraph` as a `unique-id`-affected node type.

I need `unique-id` enabled for paragraphs though, which is a combination of extensions I simply can’t make working, and I’m not sure whether I’m expected to work this out by myself (somehow using `filterTransaction`?) or if this indeed is a bug.

---

I do get the all too familiar _Uncaught RangeError: Can not convert <"https://www.youtube.com/watch?v=qF_jSMjYTm0"> to a Fragment (looks like multiple versions of prosemirror-model were loaded)_ error when pasting a recognized YouTube url into the paragraph node, but according to `yarn list --pattern prosemirror-model` I have exactly one version installed: `prosemirror-model@1.19.3`. (Not sure if the error message uses “version” correctly, or if it’s more about multiple **instances** of the dependencies or something?)

### What browser are you using?

Chrome

### What did you expect to happen?

I expect the paragraph to receive a unique ID **and** for the pasted Youtube URL to turn into an embedded video. 

### Did you update your dependencies?

- [X] Yes, I’ve updated my dependencies to use the latest version of all packages.

### Are you sponsoring us?

- [ ] Yes, I’m a sponsor. 💖

**Comment by Haraldson** [comment]
This has been broken for more than three months now, and with Pro extensions available for everyone for free, I feel like this deserves more attention.

**Comment by Haraldson** [comment]
This seems to have been fixed between 2.2.1 and 2.2.4. Not sure what did the trick, but closing this issue now.
